### PR TITLE
Add test cases to preview route

### PR DESCRIPTION
**Pull request by fmrsabino**
- Adds additional test cases to `/preview` route

### CODE REVIEW

**Review by moisses89 — COMMENTED** [review]
Looks good a few questions :)

File: src/routes/transactions/tests/preview.rs [hunk 1]
```diff
@@ -139,3 +114,100 @@ async fn post_preview_success() {
...
+    )
+    .await
+    .expect("Valid rocket instance");
+
+    let request =  client.post("/v1/chains/1/transactions/0x37D94d4E230859f83c0868CebEd8CcB83A765cee/preview")
+        .header(Header::new("Host", "test.gnosis.io"))
+        .header(ContentType::JSON)
+        .body(&json!({
+            "to": "0x37D94d4E230859f83c0868CebEd8CcB83A765cee",
+            "data": "0x095ea7b3000000000000000000000000ae9844f89d98c150f5e61bfc676d68b4921559900000000000000000000000000000000000000000000000000001c6bf52634000",
```
[the remark this answers is not in the document]
Comment (fmrsabino) [reply src/routes/transactions/tests/preview.rs:199]: I did this so we don't tie directly the expected data with the setup data.

If we refactor something around the test data it'd be nice for this to fail independently as a warning that the expected data is different (this is more to protect against accidental refactors).

File: src/routes/transactions/tests/preview.rs [hunk 2]
```diff
@@ -139,3 +114,100 @@ async fn post_preview_success() {
     assert_eq!(actual_status, Status::Ok);
```
[the remark this answers is not in the document]
Comment (moisses89) [reply src/routes/transactions/tests/preview.rs:114]: Repeated question ;)

File: src/routes/transactions/tests/preview.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (moisses89) [reply src/routes/transactions/tests/preview.rs:199]: Ok, If was something on purpose, understood.

**Review by moisses89 — APPROVED** [review]

**Pull request merged**